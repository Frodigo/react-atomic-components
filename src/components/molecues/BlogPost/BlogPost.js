import React from 'react';
import Moment from 'react-moment';
import PropTypes from 'prop-types';

import Link from '../../atoms/Link';

import classes from './BlogPost.module.css';

const BlogPost = (props) => {
    const {
        title,
        author,
        date,
        url,
        image,
        shortDescription,
        ...rest
    } = props;

    return (
        <div className={classes.block} {...rest}>
            <div className={classes.blockInner}>
                <section className={classes.text}>
                    <h2 className={classes.heading}>
                        <a href={url}>{title}</a>
                    </h2>

                    <div className={classes.meta}>
                        <span className={classes.author}>{author}</span>
                        <span className={classes.metaItem}>
                            <Moment
                                format="D MMMM YYYY"
                                date={new Date(date)}
                            />
                        </span>
                    </div>

                    <Link variant="primary" href={url}>
                        Read more
                    </Link>
                </section>
                <div className={classes.shortDescription}>
                    <p>
                        {shortDescription}{' '}
                        <a href={url}>Read more</a>
                    </p>
                </div>
                <figure className={classes.figure}>
                    <a href={url}>
                        <img src={image} alt={title} width={500} />
                    </a>
                </figure>
            </div>
        </div>
    );
};

BlogPost.propTypes = {
    author: PropTypes.string.isRequired,
    date: PropTypes.string.isRequired,
    image: PropTypes.string.isRequired,
    shortDescription: PropTypes.string.isRequired,
    title: PropTypes.string.isRequired,
    url: PropTypes.string.isRequired
};

export default BlogPost;
