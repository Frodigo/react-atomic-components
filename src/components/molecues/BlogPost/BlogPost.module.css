.block {
    display: flex;
    flex-direction: column;
    padding: 30px 0;
    border-bottom: solid 2px var(--color-text);
}

.blockInner {
    padding: 0 var(--spacing-xs);
}

.heading {
    margin-top: 0;
    margin-bottom: 12px;
}
.heading a {
    color: inherit;
    font-size: inherit;
    text-decoration: none;
}

.meta {
    margin-bottom: 12px;
}

.metaItem {
    display: inline-block;
    margin-left: 17px;
}

.author {
    composes: metaItem;
    font-weight: 700;
}

.metaItem:first-child {
    margin-left: 0;
}

.figure {
    display: none;
}

.shortDescription {
    display: none;
}


@media (min-width: 768px) {
    .meta + a {
        display: none;
    }

    .blockInner {
        align-items: center;
        display: flex;
    }

    .text,
    .figure  {
        flex: 1 1 50%;
    }

    .figure {
        display: flex;
        justify-content: flex-end;
        padding-left: var(--spacing-md);
    }
}

@media (min-width: 1300px) {
    .blockInner {
        margin: 0 auto;
        max-width: var(--desktop-container-width);
    }

    .heading a:hover,
    .heading a:focus {
        color: var(--color-secondary)
    }

    .shortDescription {
        display: block;
        padding: 0 var(--spacing-md);
    }

    .text {
        padding-right: var(--spacing-md);
    }

    .text,
    .figure,
    .shortDescription  {
        flex: 1 1 33%;
    }
}