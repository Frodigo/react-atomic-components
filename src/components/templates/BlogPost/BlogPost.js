import React from 'react';

import PageHeader from '../../organisms/PageHeader/PageHeader';
import Breadcrumbs from '../../molecues/Breadcrumbs';
import Newsletter from '../../organisms/Newsletter';
import PageFooter from '../../organisms/PageFooter/PageFooter';
import MobileNav from '../../organisms/MobileNav/MobileNav';
import { items as mobileNavItems } from '../../organisms/MobileNav/MobileNav.config';
import Heading from '../../atoms/Text/Heading';
import BlockQuote from '../../atoms/Text/Blockquote/Blockquote';
import AboutTheAuthor from '../../organisms/AboutTheAuthor';
import ShareSection from '../../organisms/ShareSection';
import Tout from '../../molecues/Tout';

import classes from './BlogPost.module.css';

const BlogPost = () => {
    const breadcrumbsItems = [
        {
            label: 'Home',
            url: '#'
        },
        {
            label: 'Blog',
            url: '#'
        },
        {
            label: 'PWA Studio',
            url: '#'
        }
    ];

    const newsletterProps = {
        title: 'Subscribe to the newsletter',
        subtitle: 'I will send you unique content every Friday.',
        submitUrl: 'https://app.getresponse.com/add_subscriber.html',
        token: 'WMdq9',
        thankYouPageUrl: 'https://marcin-kwiatkowski.com/thank-you',
        footer:
            'Also i will send you send you PDF with 112 pages about PWA Studio Rich Content renderers',
        onSubmit: (data, e) => e.target.submit()
    };

    const post = {
        author: 'John',
        date: 'November 27, 2020',
        heroImage: 'https://via.placeholder.com/320x180',
        title: 'Heading level 1'
    };

    const aboutTheAuthorProps = {
        name: 'John',
        image: 'https://via.placeholder.com/270x270',
        description:
            'Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium dui. In malesuada enim in dolor euismod, id commodo mi consectetur. Curabitur at vestibulum nisi. Nullam vehicula nisi velit. Mauris turpis nisl, molestie ut ipsum et, suscipit vehicula odio. Vestibulum interdum vestibulum felis ac molestie.',
        ctaUrl: '#',
        ctaLabel: 'Read more'
    };

    const pageHeaderNavItems = [
        {
            url: '#',
            label: 'Home',
            isActive: true
        },
        {
            url: '#',
            label: 'About me'
        },
        {
            url: '#',
            label: 'Blog'
        },
        {
            url: '#',
            label: 'Get in touch'
        }
    ];

    return (
        <>
            <PageHeader navItems={pageHeaderNavItems} />
            <div className={classes.postHeader}>
                <Heading type="h1">{post.title}</Heading>
                <div className={classes.postMetadata}>
                    {post.author} / {post.date}
                </div>
                <Breadcrumbs items={breadcrumbsItems} />
            </div>

            <figure className={classes.postHeroImage}>
                <img src={post.heroImage} alt={post.title} />
            </figure>

            <div className={classes.content}>
                <p>
                    Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium
                    dui. In malesuada enim in dolor euismod, id commodo mi
                    consectetur. Curabitur at vestibulum nisi. Nullam vehicula
                    nisi velit. Mauris turpis nisl, molestie ut ipsum et,
                    suscipit vehicula odio. Vestibulum interdum vestibulum felis
                    ac molestie.
                </p>
                <h2>Heading Level 2</h2>

                <p>
                    Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium
                    dui. In malesuada enim in dolor euismod, id commodo mi
                    consectetur. Curabitur at vestibulum nisi. Nullam vehicula
                    nisi velit. Mauris turpis nisl, molestie ut ipsum et,
                    suscipit vehicula odio. Vestibulum interdum vestibulum felis
                    ac molestie.
                </p>
                <h3>Heading Level 3</h3>

                <figure>
                    <img
                        src="https://via.placeholder.com/320x180"
                        alt="figure"
                    />
                </figure>

                <h4>Heading level 4</h4>
                <p className="paragraphSmaller">
                    Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium
                    dui. In malesuada enim in dolor euismod, id commodo mi
                    consectetur. Curabitur at vestibulum nisi. Nullam vehicula
                    nisi velit. Mauris turpis nisl, molestie ut ipsum et,
                    suscipit vehicula odio. Vestibulum interdum vestibulum felis
                    ac molestie.
                </p>

                <BlockQuote
                    author="Marcin Kwiatkowski"
                    quote="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin tempus ullamcorper ante, 
                    eu mattis est auctor id. Nunc libero nisl, efficitur at aliquet vel, venenatis eget felis. Suspendisse at diam nec 
                    risus scelerisque tempus. Mauris id eros diam. Mauris ac risus id libero laoreet eleifend in ac elit."
                />

                <p className="paragraphLarge">
                    Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium
                    dui. In malesuada enim in dolor euismod, id commodo mi
                    consectetur. Curabitur at vestibulum nisi. Nullam vehicula
                    nisi velit. Mauris turpis nisl, molestie ut ipsum et,
                    suscipit vehicula odio. Vestibulum interdum vestibulum felis
                    ac molestie.
                </p>

                <h5>Heading level 5</h5>

                <ol>
                    <li>Item 1</li>
                    <li>Item 2</li>
                    <li>Item 3</li>
                </ol>

                <h6>Heading level 6</h6>

                <ul>
                    <li>Item 1</li>
                    <li>Item 2</li>
                    <li>Item 3</li>
                </ul>
            </div>

            <AboutTheAuthor {...aboutTheAuthorProps} />
            <Newsletter {...newsletterProps} />
            <ShareSection />
            <div className={classes.relatedProducts}>
                <Heading type="h4">You may also like</Heading>
                <div className={classes.items}>
                    <Tout
                        imageUrl="https://via.placeholder.com/290x240"
                        linkUrl="#"
                        title="Lorem ipsum"
                    />
                    <Tout
                        imageUrl="https://via.placeholder.com/290x240"
                        linkUrl="#"
                        title="Lorem ipsum"
                    />
                </div>
            </div>
            <PageFooter footerColumns={[
                {
                    header: 'Column 1',
                    links: [{
                        url: '#',
                        label: 'lorem ipsum'
                    },{
                        url: '#',
                        label: 'dolor sit amet'
                    }]
                },
                {
                    header: 'Column 2',
                    links: [{
                        url: '#',
                        label: 'lorem ipsum'
                    },{
                        url: '#',
                        label: 'dolor sit amet'
                    }]
                },
                {
                    header: 'Column 3',
                    links: [{
                        url: '#',
                        label: 'lorem ipsum'
                    },{
                        url: '#',
                        label: 'dolor sit amet'
                    }]
                },
                {
                    header: 'Column 4',
                    links: [{
                        url: '#',
                        label: 'lorem ipsum'
                    },{
                        url: '#',
                        label: 'dolor sit amet'
                    }]
                }
            ]} footerBottomLinks={[
                {
                    href: '#',
                    label: 'Link 1'
                },
                {
                    href: '#',
                    label: 'Link 2'
                },
                {
                    href: '#',
                    label: 'Link 3'
                }
            ]} copyright="&copy; 2021 Lorem ipsum" />
            <MobileNav items={mobileNavItems} />
        </>
    );
};

export default BlogPost;
