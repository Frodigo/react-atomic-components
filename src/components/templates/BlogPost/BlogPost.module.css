.postHeader {
    padding: var(--spacing-md);
}

.postMetadata {
    border-bottom: 2px solid var(--color-secondary);
    margin-bottom: var(--spacing-md);
    padding: var(--spacing-xs) 0;
}

.postHeader nav {
    border-bottom: 0;
    margin-top: 5px;
    padding: 0
}

.postHeroImage {
    margin-bottom: var(--spacing-xxl);
}

.postHeroImage img {
    display: block;
    width: 100%;
}



.relatedProducts h4 {
    margin: 30px 0;
    text-align: center;
}

@media (min-width: 768px) {
    .items {
        display: flex;
        padding-left: 15px;
        padding-right: 15px;
    }

    .relatedProducts {
        padding-bottom: var(--spacing-xxl);
    }

    .items > div {
        flex: 1 1 50%;
        margin: 0 var(--spacing-md);
    }
}

@media (min-width: 1300px) {
    .postHeader,
    .postHeroImage {
        margin: 0 auto;
        max-width: var(--small-desktop-container-width);
        padding-left: 0;
        padding-right: 0;
    }

    .postHeader {
        margin-top: 40px;
    }

    .content {
       margin-top: 40px;

    }

    .content > div:first-child {
        max-width: var(--small-desktop-container-width);
    }

    .relatedProducts {
        max-width: var(--desktop-container-width);
        margin: 60px auto;
    }
}
