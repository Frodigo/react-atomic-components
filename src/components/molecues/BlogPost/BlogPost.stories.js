import React from 'react';

import { default as BlogPostComponent } from './BlogPost';

export default {
    title: 'Molecues',
    component: BlogPostComponent
};

const Template = (args) => <BlogPostComponent {...args} />;

export const BlogPost = Template.bind({});
BlogPost.args = {
    title: 'Lorem ipsum dolor sit amet, vel maximus ante faucibus sed.',
    author: 'Jan Kowalski',
    date: 'October 10, 2020',
    url: '#',
    image: 'https://via.placeholder.com/396x222',
    shortDescription: 'Quisque rutrum ac lectus ullamcorper placerat. Morbi condimentum malesuada dui, id consectetur lectus dictum vel. Integer bibendum mauris id nisl dignissim, ut lacinia dolor commodo. Sed eros nisl, interdum hendrerit felis ut, pellentesque mattis eros. '
};
