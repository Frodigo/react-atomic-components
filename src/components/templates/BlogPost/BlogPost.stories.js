import React from 'react';

import { default as BlogPostComponent } from './BlogPost';

export default {
    title: 'Templates',
    component: BlogPostComponent
};

const Template = (args) => <BlogPostComponent {...args} />;

export const BlogPost = Template.bind({});
