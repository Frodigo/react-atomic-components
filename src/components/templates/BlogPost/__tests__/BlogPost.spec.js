import React from 'react';
import { render } from '@testing-library/react';

import BlogPost from '../BlogPost';

describe('Blog post should be rendered', () => {
    test('Blog post renders correctly', () => {
        const { container } = render(<BlogPost />);

        expect(container).toMatchSnapshot();
    });
});
