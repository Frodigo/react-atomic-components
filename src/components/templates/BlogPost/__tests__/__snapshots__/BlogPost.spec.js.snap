// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog post should be rendered Blog post renders correctly 1`] = `
<div>
  <header
    class="pageHeader"
  >
    <div
      class="container"
    >
      <a
        class="link"
        href="/"
      >
        <img />
      </a>
      <nav
        class="primaryNav"
      >
        <a
          class="navLinkActive navLink"
          href="#"
        >
          Home
        </a>
        <a
          class=" navLink"
          href="#"
        >
          About me
        </a>
        <a
          class=" navLink"
          href="#"
        >
          Blog
        </a>
        <a
          class=" navLink"
          href="#"
        >
          Get in touch
        </a>
      </nav>
    </div>
  </header>
  <div
    class="postHeader"
  >
    <h1
      class=""
    >
      Heading level 1
    </h1>
    <div
      class="postMetadata"
    >
      John
       / 
      November 27, 2020
    </div>
    <nav
      class="breadcrumbs"
      items="[object Object],[object Object],[object Object]"
    >
      <div
        class="container"
      >
        <a
          class="breadcrumbsItem"
          href="#"
        >
          <span
            class="label"
          >
            Home
          </span>
        </a>
        <a
          class="breadcrumbsItem"
          href="#"
        >
          <span
            class="label"
          >
            Blog
          </span>
        </a>
        <span
          class="activeItem"
        >
          PWA Studio
        </span>
      </div>
    </nav>
  </div>
  <figure
    class="postHeroImage"
  >
    <img
      alt="Heading level 1"
      src="https://via.placeholder.com/320x180"
    />
  </figure>
  <div
    class="content"
  >
    <p>
      Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium dui. In malesuada enim in dolor euismod, id commodo mi consectetur. Curabitur at vestibulum nisi. Nullam vehicula nisi velit. Mauris turpis nisl, molestie ut ipsum et, suscipit vehicula odio. Vestibulum interdum vestibulum felis ac molestie.
    </p>
    <h2>
      Heading Level 2
    </h2>
    <p>
      Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium dui. In malesuada enim in dolor euismod, id commodo mi consectetur. Curabitur at vestibulum nisi. Nullam vehicula nisi velit. Mauris turpis nisl, molestie ut ipsum et, suscipit vehicula odio. Vestibulum interdum vestibulum felis ac molestie.
    </p>
    <h3>
      Heading Level 3
    </h3>
    <figure>
      <img
        alt="figure"
        src="https://via.placeholder.com/320x180"
      />
    </figure>
    <h4>
      Heading level 4
    </h4>
    <p
      class="paragraphSmaller"
    >
      Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium dui. In malesuada enim in dolor euismod, id commodo mi consectetur. Curabitur at vestibulum nisi. Nullam vehicula nisi velit. Mauris turpis nisl, molestie ut ipsum et, suscipit vehicula odio. Vestibulum interdum vestibulum felis ac molestie.
    </p>
    <figure
      class="figure"
    >
      <div
        class="inner"
      >
        <blockquote
          class="quote"
        >
          <span>
            svg
          </span>
          <p
            class="paraghaph"
            style="text-align: center;"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin tempus ullamcorper ante,  eu mattis est auctor id. Nunc libero nisl, efficitur at aliquet vel, venenatis eget felis. Suspendisse at diam nec  risus scelerisque tempus. Mauris id eros diam. Mauris ac risus id libero laoreet eleifend in ac elit.
          </p>
        </blockquote>
        <figcaption
          class="caption"
        >
          Marcin Kwiatkowski
        </figcaption>
      </div>
    </figure>
    <p
      class="paragraphLarge"
    >
      Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium dui. In malesuada enim in dolor euismod, id commodo mi consectetur. Curabitur at vestibulum nisi. Nullam vehicula nisi velit. Mauris turpis nisl, molestie ut ipsum et, suscipit vehicula odio. Vestibulum interdum vestibulum felis ac molestie.
    </p>
    <h5>
      Heading level 5
    </h5>
    <ol>
      <li>
        Item 1
      </li>
      <li>
        Item 2
      </li>
      <li>
        Item 3
      </li>
    </ol>
    <h6>
      Heading level 6
    </h6>
    <ul>
      <li>
        Item 1
      </li>
      <li>
        Item 2
      </li>
      <li>
        Item 3
      </li>
    </ul>
  </div>
  <div
    class="aboutTheAuthor"
  >
    <div
      class="inner"
    >
      <figure
        class="figureDesktop"
        style="background-image: url(https://via.placeholder.com/270x270);"
      />
      <article
        class="content"
      >
        <h6
          class=""
        >
          About the author
        </h6>
        <p
          class="authorName"
        >
          John
        </p>
        <figure
          class="figure"
          style="background-image: url(https://via.placeholder.com/270x270);"
        />
        <p
          class="description"
        >
          Nam porttitor blandit accumsan. Ut vel dictum sem, a pretium dui. In malesuada enim in dolor euismod, id commodo mi consectetur. Curabitur at vestibulum nisi. Nullam vehicula nisi velit. Mauris turpis nisl, molestie ut ipsum et, suscipit vehicula odio. Vestibulum interdum vestibulum felis ac molestie.
        </p>
        <a
          class="secondary"
          href="#"
        >
          Read more
        </a>
      </article>
    </div>
  </div>
  <form
    action="https://app.getresponse.com/add_subscriber.html"
    class="newsletter"
    id="newsletter"
    method="post"
  >
    <div
      class="newsletterContent"
    >
      <h3
        class="title"
      >
        Subscribe to the newsletter
      </h3>
      <p
        class="paraghaph"
      >
        I will send you unique content every Friday.
      </p>
      <div
        class="fieldset"
      >
        <div
          class="field"
        >
          <input
            class="input null"
            data-testid="input"
            name="first_name"
            placeholder="Your name"
            type="text"
            value=""
          />
        </div>
        <div
          class="field"
        >
          <input
            class="input null"
            data-testid="input"
            name="email"
            placeholder="Your email"
            type="email"
            value=""
          />
        </div>
      </div>
      <input
        name="campaign_token"
        type="hidden"
        value="WMdq9"
      />
      <input
        name="thankyou_url"
        type="hidden"
        value="https://marcin-kwiatkowski.com/thank-you"
      />
      <input
        name="start_day"
        type="hidden"
        value="0"
      />
      <footer
        class="footer"
      >
        <p>
          Also i will send you send you PDF with 112 pages about PWA Studio Rich Content renderers
        </p>
        <button
          class="primary"
          type="submit"
        >
          Subscribe
        </button>
      </footer>
    </div>
  </form>
  <div
    class="shareSection"
  >
    <h4
      class=""
    >
      Did you like it? Share!
    </h4>
  </div>
  <div
    class="relatedProducts"
  >
    <h4
      class=""
    >
      You may also like
    </h4>
    <div
      class="items"
    >
      <div
        class="tout"
      >
        <a
          href="#"
        >
          <figure
            class="figure"
          >
            <img
              alt="figure"
              layout="responsive"
              src="https://via.placeholder.com/290x240"
            />
          </figure>
          <header
            class="tape"
          >
            <h5
              class="inverse"
            >
              Lorem ipsum
            </h5>
          </header>
        </a>
      </div>
      <div
        class="tout"
      >
        <a
          href="#"
        >
          <figure
            class="figure"
          >
            <img
              alt="figure"
              layout="responsive"
              src="https://via.placeholder.com/290x240"
            />
          </figure>
          <header
            class="tape"
          >
            <h5
              class="inverse"
            >
              Lorem ipsum
            </h5>
          </header>
        </a>
      </div>
    </div>
  </div>
  <footer
    class="pageFooter"
  >
    <div
      class="columns"
    >
      <div
        class="footerColumn"
      >
        <button
          class="header"
          type="button"
        >
          <h5
            class="heading"
          >
            Column 1
          </h5>
          <span
            class="expandButton"
          >
            <span>
              svg
            </span>
          </span>
        </button>
      </div>
      <div
        class="footerColumn"
      >
        <button
          class="header"
          type="button"
        >
          <h5
            class="heading"
          >
            Column 2
          </h5>
          <span
            class="expandButton"
          >
            <span>
              svg
            </span>
          </span>
        </button>
      </div>
      <div
        class="footerColumn"
      >
        <button
          class="header"
          type="button"
        >
          <h5
            class="heading"
          >
            Column 3
          </h5>
          <span
            class="expandButton"
          >
            <span>
              svg
            </span>
          </span>
        </button>
      </div>
      <div
        class="footerColumn"
      >
        <button
          class="header"
          type="button"
        >
          <h5
            class="heading"
          >
            Column 4
          </h5>
          <span
            class="expandButton"
          >
            <span>
              svg
            </span>
          </span>
        </button>
      </div>
    </div>
    <div>
      <ul
        class="bottomNav"
      >
        <li>
          <a
            href="#"
          >
            Link 1
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Link 2
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Link 3
          </a>
        </li>
      </ul>
      <p
        class="copyright"
      >
        © 2021 Lorem ipsum
      </p>
    </div>
  </footer>
  <nav
    class="mobileNav"
  >
    <a
      href="/"
    >
      <span
        class="iconContainer"
      >
        <img
          alt="Home"
          src="[object Object]"
        />
        <span
          class="label"
        >
          Home
        </span>
      </span>
    </a>
    <a
      href="/blog"
    >
      <span
        class="iconContainer"
      >
        <img
          alt="Blog"
          src="[object Object]"
        />
        <span
          class="label"
        >
          Blog
        </span>
      </span>
    </a>
    <a
      href="/about-me"
    >
      <span
        class="iconContainer"
      >
        <img
          alt="About"
          src="[object Object]"
        />
        <span
          class="label"
        >
          About
        </span>
      </span>
    </a>
    <a
      href="/get-in-touch"
    >
      <span
        class="iconContainer"
      >
        <img
          alt="Contact"
          src="[object Object]"
        />
        <span
          class="label"
        >
          Contact
        </span>
      </span>
    </a>
    <a
      href="#newsletter"
    >
      <span
        class="iconContainer"
      >
        <img
          alt="Subscribe"
          src="[object Object]"
        />
        <span
          class="label"
        >
          Subscribe
        </span>
      </span>
    </a>
  </nav>
</div>
`;
